third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -60,7 +60,7 @@ PjRtComputationClient::PjRtComputationClient() {
     client_ = xla::GetTpuClient(max_inflight_computations).value();
   } else if (device_type == "TPU_C_API") {
     TF_VLOG(1) << "Initializing PjRt C API client...";
-    client_ = std::move(xla::GetCApiClient().value());
+    client_ = std::move(xla::GetCApiClient(device_type).value());
```

Comment:
I think the canonical device type should be "TPU" instead of "TPU_C_API"

Reply:
`TPU_C_API` is a special mode to test the new CAPI bridge.

Reply:
`device_type` check should be against "TPU_C_API", but the call to `xla::GetCApiClient()` I think we should pass "TPU".

Reply:
Thanks, updated to "TPU"

Reply:
Yea, for a future reference, tpu initializer would do something like,
```
stream_executor::tpu::SetPjrtApi("TPU", GetTpuPjrtApi()).IgnoreError();
```
and the "TPU" is used as a key (lower-cased) to map the C API to the device type. cc @will-cromar @JackCaoG

Reply:
should we lower-case "TPU" right now or keep "TPU" and lower-cased by function?

Reply:
It's done by the function.

Reply:
Thanks, Yeounoh!